lib/interface_quda.cpp

```diff
@@ -5356,6 +5443,160 @@ void performWuppertalnStep(void *h_out, void *h_in, QudaInvertParam *inv_param,
 
   profileWuppertal.TPSTOP(QUDA_PROFILE_TOTAL);
 }
+ 
+
+void performTwoLinkGaussianSmearNStep(void *h_in, QudaInvertParam *inv_param, const int n_steps, const double width, const int compute_2link = 1, const int delete_2link = 0, const int t0 = -1)
```

Comment:
It would be better to collect the parameters for this function into either a new struct , or to augment the `QudaInvertParam` with these new parameters.

Reply:
Moreover, placing the parameters into the `QudaInvertParam` struct would allow for defaults to be set for these parameteres.

Reply:
done